package.json

```diff
@@ -31,7 +31,7 @@
     "mkdirp": "^0.5.1",
     "napi-build-utils": "^1.0.1",
     "node-abi": "^2.2.0",
-    "node-gyp": "^3.0.3",
+    "node-gyp": "^5.0.7",
```

Comment:
Let's go straight to 6x.

Reply:
Only reason I didn't go straight to 6.x is that the `latest` dist tag for node-gyp on npm is 5.x.

Reply:
Hm OK. I asked why that is: https://github.com/nodejs/node-gyp/pull/2013#issuecomment-571229235

Reply:
my bad, it was supposed to be `latest`, the only major diff for 6.x is that it will pick `python3` over `python2` where it exists on the system, otherwise the two are in sync, so 👍 to going straight to 6. We also have a possible 7 coming out soon too, but we're still maintaining 5 for as long as npm keeps on using it.